package.json

```diff
@@ -17,6 +17,7 @@
     "build": "rm -rf dist && NODE_ENV=development webpack",
     "build:staging": "rm -rf dist && NODE_ENV=staging webpack",
     "build:production": "rm -rf dist && NODE_ENV=production webpack",
+    "start": "npm dist",
```

Comment:
What’s this for?

Reply:
nothing. was going to make it a node server but forgot we're just serving an index.html file so switched to nginx.